WookieeMatt/Essence20
# Creating upgrades subcontainers

##### In this PR
- Adding upgrade subcontainers within the weapon and armor containers
- Recursive templates did not seem to work, so I tried to minimize duplication between container.hbs and subcontainer.hbs by breaking things up more
- Supporting drops, + button, deletes, and accordion toggle
- Creating a new `UpgradeHandler` for upgrade-related actor sheet stuff
- Using `<hr>`s + flex gaps instead of bottom borders to separate items, this way we don't have to mess with margins everywhere
- Dropping upgrades when you have multiple weapons/armors will prompt you to select one

##### Testing
- Create a PC and add weapons and armor
- Add upgrades via + and drag/drop
- If you have a single weapon/armor and drop an upgrade, it should be added automatically
- If you have multiple weapons/armors, a dialog will prompt you to select one
- Deleting and toggling should work
